### ISSUE TITLE
environment variables with char "(" break the deploy

### ISSUE THREAD

**Issue by cezarsa** [tsuru/tsuru#1089]
This happens because the deploy command include envs in the command line, and this envs aren't being properly escaped.
```
$ tsuru env-set A="a(a"
$ tsuru app-deploy . -a gtest
Uploading files... ok
/bin/bash: -c: line 0: syntax error near unexpected token `('
/bin/bash: -c: line 0: `if [[ $(tsuru_unit_agent --help | head -n1 | grep deploy) ]]; then tsuru_unit_agent http://192.168.50.4:8080 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx gtest "/var/lib/tsuru/deploy archive file:///home/application/archive.tar.gz" deploy; else /var/lib/tsuru/deploy archive file:///home/application/archive.tar.gz TSURU_APPNAME=gtest TSURU_APPDIR=/home/application/current TSURU_HOST=http://192.168.50.4:8080 TSURU_APP_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx A=a(a ; fi'
Exit status 1
```

**Comment by cezarsa** [comment]
Also, with `tsuru env-set A="a\"a"` breaks with:
```
/bin/bash: -c: line 0: unexpected EOF while looking for matching `"'
/bin/bash: -c: line 1: syntax error: unexpected end of file
```

**Comment by cezarsa** [comment]
Fixed by 5c06d2e48ed6533a15a48b0977e1348eb3b2341e and tsuru/tsuru-unit-agent@182e3a6fbfdfddc719df395a7f2c13a4b98653be
